### PR TITLE
[unit tests] add more geometry unittests

### PR DESCRIPTION
**Pull request by mykola-f**
Added at least some test for every function in geometry.cpp

Signed-off-by: Mykola Faryma <myk.faryma@gmail.com>

### CODE REVIEW

File: tests/geometry_ut.cpp [hunk 1]
```diff
@@ -4,4 +4,68 @@
 TEST(EuqlidianDistance, EuqlidianDistance)
 {
     EXPECT_EQ(EuqlidianDistance({ cv::Point{5, 5}, cv::Point{10, 15} }), 11);
+}
+
+TEST(getNormal, getNormal)
+{
+	Line expected = { cv::Point{-3, 15}, cv::Point{7, 10}};
```
Comment (Blyschak) [inline tests/geometry_ut.cpp]: tab and spaces mixed

File: tests/geometry_ut.cpp [hunk 2]
```diff
@@ -4,4 +4,68 @@
...
+{
+	Line expected = {{ cv::Point{400, 300}, cv::Point{400, -2147483648} }};
+	auto result = getTangentToCircle(cv::Point{400, 300}, cv::Point{300, 300}, 100);
+    EXPECT_EQ(result, expected);
+}
+
+TEST(getInnerAngleBetweenLines, getInnerAngleBetweenLines)
+{
+	double expected = 90.0;
+	auto result = getInnerAngleBetweenLines({{ cv::Point{0, 5}, cv::Point{0, 0} }},
```
Comment (Blyschak) [inline tests/geometry_ut.cpp]: general q: does this func really return inner angle?

**Comment by Blyschak** [comment]
LGTM, will apply uncrsutify later

File: tests/geometry_ut.cpp (hunk 2 above)
Comment (mykola-f) [reply tests/geometry_ut.cpp]: Well, it returns an angle :)
I guess I'll open an issue to refactor it.

File: tests/geometry_ut.cpp (hunk 1 above)
Comment (mykola-f) [reply tests/geometry_ut.cpp]: @Blyschak I suggest to turn on uncrustify asap

**Pull request merged**